lib/core/device_drivers.rb

```diff
@@ -179,22 +179,19 @@ def merge_chrome_ops(config_caps, case_caps)
 
   # build the Chrome driver, given a set of options
   def build_chrome_driver(chrome_ops)
+    # local selenium instance
+    chromeOptions = Selenium::WebDriver::Remote::Capabilities.chrome(
```

Comment:
Why are these not `Selenium::WebDriver::Chrome::Options` like in my example? Like I said, `Selenium::WebDriver::Remote::Capabilities` is deprecated and should be removed.

Reply:
The way of adding prefs and args is a bit... well, not as simple, but I have changed it

Reply:
Checked the latest changes. If the `chrome_ops`/`firefox_ops` etc. are only expected to have the `args`/`prefs`/`profile` keys, it can be simplified with the splat operator:
```ruby
ops = Selenium::WebDriver::Chrome::Options.new(**chrome_opts)
 ```
 [See here for more details about it](https://ruby-doc.org/core-2.0.0/doc/syntax/calling_methods_rdoc.html#label-Hash+to+Keyword+Arguments+Conversion).

Reply:
Yeah, good point, Ill add that for all of them, it makes more sense

Reply:
Just commited a change that solves this and adds the remote driver helper definition